src/manifest/indexes/timeline.ts

```diff
@@ -304,10 +304,10 @@ const SegmentTimelineHelpers: ISegmentHelpers<ITimelineIndex> = {
       duration : number;
       timescale : number;
     },
-    currentSegment : {
+    currentSegment? : {
       time : number;
-      duration : number;
-      timescale : number;
+      duration? : number;
+      timescale? : number;
```

Comment:
In which case would timescale be undefined?

Without timescale, time and duration should be unexploitable